templates/multiclusterhub/base/rbac/clusterrole-edit-aggregate.yaml

```diff
@@ -8,8 +8,11 @@ metadata:
   name: open-cluster-management:edit-aggregate
 rules:
 - apiGroups: ["cluster.open-cluster-management.io"]
-  resources: ["managedclusters", "managedclustersets", "managedclusters/status", "clustercurators"]
+  resources: ["managedclusters", "managedclusters/status", "clustercurators"]
```

Comment:
I think we should remove all cluster scoped resource in admin/editor/view clusterrole. These clusterrole are mainly for ns scoped resource. WDYT  @chaitanyaenr ?